### PR TITLE
Android: update translations for 0.5.2

### PR DESCRIPTION
**Pull request by carstene1ns**

### CODE REVIEW

**Comment by carstene1ns** [comment]
There has been a recent change in the Chinese translation. Do not merge yet.

**Comment by carstene1ns** [comment]
Added now, ready to merge.

File: builds/android/app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -56,5 +75,12 @@
   <string name="reset_button_mapping">重置按钮</string>
   <string name="exit_without_saving">不保存并退出</string>
   <string name="save_and_quit">保存并退出</string>
+  <string name="key_enter">确定(Z)</string>
+  <string name="key_cancel">取消(ESC或X)</string>
+  <string name="key_shift">S</string>
```
Comment (fdelapena) [inline builds/android/app/src/main/res/values-zh/strings.xml]: This one may be a bit confusing (S for shift key), I was expecting 换档键

Comment (carstene1ns) [inline builds/android/app/src/main/res/values-zh/strings.xml]: So should we use `换档键(Shift)`?

Comment (fdelapena) [inline builds/android/app/src/main/res/values-zh/strings.xml]: I was thinking on this approach precisely, looks good to me

**Pull request merged**